### PR TITLE
Bugfix on period controler/view

### PR DESCRIPTION
**Pull request by symartin**
# Description

- Fix the bug preventing to assigne a beneficiary on a period slot (on Firefox)
- Fix the bug that let user to add a beneficiary linked to a withdrawn member to a period slot
- Fix the CSS breadcrumb icons (affect the whole website)
- Make the UI more clear on the period page (better differentiation on what can be clicked)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] The assets has been changed

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] On development environment
- [x] On preprod environement
- [x] On desktop screen size
- [x] On mobile screen size

## Preprod test configuration:
* PHP version:
* OS:
    - Distributor ID: Ubuntu  
    - Description: Ubuntu 22.04 LTS  
    - Release: 22.04  
    - Codename: jammy
* Browser:
    - [x] Firefox desktop
    - [x] Edge
    - [x] Chrome desktop

* Database server: Maria DB

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/AppBundle/Resources/public/css/custom.less [hunk 1]
```diff
@@ -8,47 +8,56 @@
 @import "card";
```
Comment (raphodn) [inline src/AppBundle/Resources/public/css/custom.less:8]: t'as fait passé un linter sur ce fichier ? ^^

**Review by raphodn — APPROVED** [review]
looks good ! (je n'ai pas testé en local)

File: app/Resources/views/admin/period/edit.html.twig [hunk 2]
```diff
@@ -86,7 +86,7 @@
                                     </div>
                                     <div class="col s3">
                                         <input type="hidden" class="checkedTypeService{{ position.id }}" name="typeService{{ position.id }}" value=0>
-                                        <button class="btn add" onclick="postPosition({{ position.id }})">Ajouter</button>
+                                        <button type="button" class="btn add" onclick="postPosition({{ position.id }})">Ajouter</button>
```
[the remark this answers is not in the document]
Comment (symartin) [reply app/Resources/views/admin/period/edit.html.twig:89]: oui par défaut, dans Firefox les buttons sont de type 'submit'...

File: src/AppBundle/Resources/public/css/custom.less (hunk 1 above)
Comment (symartin) [reply src/AppBundle/Resources/public/css/custom.less:8]: j'ai modifié qq lignes pour aligner les icons des breadcrumbs. Mais oui tent qu'a faire, j'ai utilisé le linter de mon IDE. c'est qd mm plus facile à lire XD

**Pull request merged**